geoportal-connectors/geoportal-harvester-waf/src/main/java/com/esri/geoportal/harvester/waf/ContentAnalyzer.java

```diff
@@ -47,11 +49,13 @@ public ContentAnalyzer(URL root) {
       if (quotMatcher.find()) {
         String extractedUrl = quotMatcher.group().replaceAll("^.|.$", "").replaceAll("\\{", "%7B").replaceAll("\\}", "%7D");
         try {
+          extractedUrl =  StringEscapeUtils.unescapeHtml4(extractedUrl);
```

Comment:
Looks like the most recent version from 'master' branch has it fixed already, although in a slightly different way.